nodes/widgets/ui_chart.html

```diff
@@ -103,8 +104,8 @@
                     }
                 },
                 yAxisLabel: { value: '' },
-                yAxisProperty: { value: null },
-                yAxisPropertyType: { value: 'property' },
+                yAxisProperty: { value: 'payload' },
+                yAxisPropertyType: { value: 'msg' },
```

Comment:
Is changing the default to msg.payload sane?

I am struggling to grasp why?

Reply:
I think so, the first/easiest use case for a chart is to use a numerical value coming in via `msg.payload` (e.g. a slider) with the x-value being generated as a `timestamp`

Reply:
Similarly, in a case where they have sensor data via MQTT, the `payload` may be a raw value, or an object, at which point `msg.payload.sensor` (or equivalent) is appropriate too.